SpyGame/src/edu/cpp/cs/cs141/final_proj/MoveStatus.java

```diff
@@ -9,7 +9,7 @@
 	 * Describes the result of the attempted movement.
 	 */
 	public enum MOVE_RESULT {
-		ILLEGAL, LEGAL, UNMOVED, WIN, LOSE, POWERUP
+		ILLEGAL, LEGAL, UNMOVED, WIN, LOSE, POWERUP;

```

Comment:
I thought it was part of the syntax, but it seems to work the same way with or without the semicolon so I guess there is no point to it. I'd just add it in for consistency since most lines of code end with a semicolon, but idk if this is part of some style standards or not.